test/functional/ui/drop_spec.lua

```diff
@@ -0,0 +1,80 @@
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local clear, feed, execute = helpers.clear, helpers.feed, helpers.execute
+
+describe(":drop", function()
+  local screen
+
+  before_each(function()
+    clear()
+    screen = Screen.new(35, 10)
+    screen:attach()
+    screen:set_default_attr_ignore({{bold=true, foreground=Screen.colors.Blue}})
+    screen:set_default_attr_ids({
+      [1] = {bold = true, reverse = true},
+      [2] = {reverse = true},
+      [3] = {bold = true},
+    })
+    execute("set laststatus=2")
+  end)
+
+  after_each(function()
+    screen:detach()
+  end)
+
+  it("works like :e when called at the start of a session", function()
```

Comment:
Nitpick: "sessions" are something else in Vim. Maybe "works like :e when not shown in any window" is better.